ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +91,75 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        title="Instructions to add Tasks or Pipelines:"
+        isOpen={isInstructionModallOpen}
+        onClose={() => setInstructionModalOpen(false)}
+        width={'50%'}
+      >
+        <Grid>
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              There are two ways to add Tasks or Pipelines on Hub
+            </Text>
+            <TextList component={TextListVariants.ol}>
+              <TextListItem>
+                <Text component={TextVariants.h5}> Add your own catalog</Text>
+                <TextList>
+                  <TextListItem>
+                    <Text>
+                      This{' '}
+                      <Text
+                        component={TextVariants.a}
+                        target="_blank"
+                        href="https://github.com/tektoncd/hub/blob/main/docs/ADD_NEW_CATALOG.md"
+                      >
+                        docs
+                      </Text>{' '}
+                      defines the steps to add your own catalog on Hub
+                    </Text>
+                  </TextListItem>
+                </TextList>
+              </TextListItem>
+              <TextListItem>
+                <Text component={TextVariants.h5}>
+                  Add your task or pipeline through{' '}
+                  <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://github.com/tektoncd/catalog"
+                  >
+                    tekton catalog{' '}
+                  </Text>
+                  and needs to follow these steps
+                </Text>
+                <TextList>
+                  <TextListItem>
+                    <Text>
+                      Prepare structure of your task or pipeline according to this{' '}
```

Comment:
Create a pull request to Tekton Catalog with Task or Pipeline with your own following this guidelines(link to the readme where this are defined)

Reply:
```
Create a pull request to Tekton Catalog(link to tektoncd/catalog) with your Task or Pipeline which must be following the guidelines mentioned here(link to the readme where this are defined)
```